### PR TITLE
Microsoft.PIX version 21.8.18.1

### PR DESCRIPTION
**Pull request by Devaniti**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37463)

### CODE REVIEW

**Comment by Devaniti** [comment]
Test in sandbox runs just fine for me
No idea what exactly makes auto tests fail, I can't find any error message apart from that it's just failed
image

File: manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 21.8.18.1
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.installer.yaml:7]: This is probably why its failing
```suggestion
- Architecture: x64
```
image

File: manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 21.8.18.1
+Installers:
+- Architecture: x86
+  InstallerType: burn
+  InstallerUrl: https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWJymh
+  InstallerSha256: CA3098327B061077F16311AD016F22761E69A66964A227192D924A7C2BC4B366
+  InstallerSwitches:
+    SilentWithProgress: /passive
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.installer.yaml:12]: InstallerSwitches are not needed when the InstallerType is specified
```suggestion
```

File: manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 21.8.18.1
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+Author: Microsoft
+PackageName: PIX on Windows
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.locale.en-US.yaml:9]: This needs to match `A`dd or `R`emove `P`rograms
```suggestion
PackageName: Microsoft PIX
```

File: manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Publisher: Microsoft Corporation
+Author: Microsoft
+PackageName: PIX on Windows
+PackageUrl: https://devblogs.microsoft.com/pix/download/
+License: Copyright (c) Microsoft Corporation. All rights reserved.
+ShortDescription: Performance tuning and debugging for DirectX 12 games on Windows
+Moniker: PIX
+Tags:
+- D3D12
+- DXR
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.locale.en-US.yaml:16]: These should be lower-case for consistent sake
```suggestion
Moniker: pix
Tags:
- d3d12
- dxr
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by Devaniti** [comment]
Thanks, setting architecture to x64 fixed the issue. I got confused since installer exe is x86 but the app itself is x64.

**Review by OfficialEsco — APPROVED** [review]
Yeah we’ve encountered quite a few of those lately.. Not like x86 is critical anyways 😁
Didn’t help that it installs to Programfiles (x86) 🤦‍♂️

**Pull request merged**